doc/network_scenarios.md

```diff
@@ -0,0 +1,418 @@
+Introduction
+============
+
+This document tries to clarify some doubts and expectations related to AutoYast network
+configuration, specially will try to show how affect the `keep_install_network` option.
+
+We put an special attemption to that flag because there was a bug in SP1 which always 
```

Comment:
I would say something like *which caused the network configuration being saved always* (the bug didn't save the network :) ).